packages/components/src/styles/components/app-frame.scss

```diff
@@ -7,6 +7,11 @@
 // APP-FRAME
 //
 
+// TODO: Create app-header tokens:
+:root {
+  --token-app-header-height: 60px;
```

Comment:
I wanted to confirm what pattern should be followed for token naming before adding tokens.